app/models/training_module.rb

```diff
@@ -55,15 +55,15 @@ def self.reload_module(slug:)
     # and can load slides for brand-new modules.
     TrainingLibrary.flush
     TrainingLibrary.load
-    self.load(slug: slug)
+    load(slug: slug)
```

Comment:
Whoops. I thought `load` was a Ruby keyword (in which case `self.` would be necessary), but it's actually just a method on `Kernel`, so it's just being overridden by the `load` method in this file. Sorry about that!